### DISCUSSION TITLE
Migrating to v2.0

### DISCUSSION THREAD

**Discussion by pintermor9** [Pycord-Development/pycord#758] [Q&A] [answered]
Will you do a migration documentation for 2.x like migrating to 1.x? If yes when?

**Comment by Lulalaby** [comment, marked as answer]
Yes
Some day :p  (On Release)

**Reply by pintermor9** [reply]
Any predections on release date?

**Reply by Lulalaby** [reply]
Just join our guild :)

https://discord.gg/pycord?event=909894372596785172

**Reply by pintermor9** [reply]
Thanks
